configure.ac

```diff
@@ -4404,14 +4404,42 @@ WITH_SAVE_ENV([
       GDBM_LIBS=${GDBM_LIBS-"-lgdbm"}
     ], [have_gdbm=no])
   ], [have_gdbm=no])
-])
 
-dnl check for _dbmmodule.c dependencies
-dnl ndbm, gdbm_compat, libdb
-AC_CHECK_HEADERS([ndbm.h], [
-  WITH_SAVE_ENV([
+  dnl check for _dbmmodule.c dependencies
+  dnl ndbm, gdbm_compat, libdb
+  AC_CHECK_HEADERS([ndbm.h], [
```

Comment:
I think we're fine keeping this inside the `GDBM_*` env flag context, since this check also encompasses GDBM compat.